roles/provision-data-sync-apb/files/data-sync-dashboard.json

```diff
@@ -108,7 +108,7 @@
       "tableColumn": "Value",
       "targets": [
         {
-          "expr": "sum(process_resident_memory_bytes{service=\"data-sync-server\"})",
+          "expr": "sum(process_resident_memory_bytes{app=\"nodejs-ex\"})",
```

Comment:
Question: Will this `nodejs-ex` value work a sync app?

Reply:
Nope. We're gonna take care of that in the card I mentioned in the description.